src/IISIntegration/src/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -444,6 +445,7 @@ IN_PROCESS_APPLICATION::SetEnvironmentVariablesOnWorkerProcess()
         m_pConfig->QueryBasicAuthEnabled(),
         m_pConfig->QueryAnonymousAuthEnabled(),
         QueryApplicationPhysicalPath().c_str(),
+        BindingInformation::Format(m_pConfig->QueryBindings()).c_str(),
```

Comment:
Why are these being set as environment variables?

Reply:
We have a common environment variable population code that is used for both In and OutOf proc